### PR TITLE
Add test models that don't use the position attribute

### LINKED ISSUE #407: Add assets with no POSITION attribute
One can argue this is a negative test, but since https://github.com/KhronosGroup/glTF/issues/1332 is asking for this, we can make an exception similar to the compatibility tests.

### PR DESCRIPTION
**Pull request by stevk**
[Mesh_NoPosition readme](https://github.com/stevk/glTF-Asset-Generator/tree/noPositions/Output/Negative/Mesh_NoPosition)

Fixes #407

### CODE REVIEW

File: Source/Property.cs [hunk 1]
```diff
@@ -106,6 +106,8 @@ static object Process(object obj)
 
     internal enum PropertyName
     {
+        Indices,
+        Position,
```
Comment (bghgary) [inline Source/Property.cs]:
```suggestion
        VertexPosition,
```

File: Source/Property.cs [hunk 2]
```diff
@@ -106,6 +106,8 @@ static object Process(object obj)
 
     internal enum PropertyName
     {
+        Indices,
```
Comment (bghgary) [inline Source/Property.cs]: How is this different than IndicesValues?

Comment (stevk) [reply Source/Property.cs]: There isn't a difference. I'll switch to IndicesValues.

File: Source/Property.cs [hunk 3]
```diff
@@ -106,8 +106,7 @@ static object Process(object obj)
 
     internal enum PropertyName
     {
-        Indices,
-        Position,
+        VertexPosition,
```
Comment (bghgary) [inline Source/Property.cs]: Move together with the other Vertex properties.

**Pull request merged**